src/Combobox/index.scss

```diff
@@ -97,3 +97,17 @@
     transform: translate(rem(1px), -50%);
   }
 }
+
+.nds-combobox-list-heading--bold {
```

Comment:
I'm using `!important` on font properties in these rulesets because the legacy `.nds-typography` class may be used in `banking`, which requires a higher specificity to override default styles on heading tags.